### DISCUSSION TITLE
Generating package from scratch

### DISCUSSION THREAD

**Discussion by jcosham** [ateucher/pkg-dev-dfo-2024-11#32] [Q&A]
Hi, 

I took this morning to go through yesterday's steps and try my hand at creating a package for some of my existing code - I thought it would be a good way to really commit (pun intended) everything to memory. I have a package with a license, description file etc., and the start of a function. I do not see any README file however - trying to remember how we generated these yesterday. I don't see it anywhere in the slides.

**Comment by ateucher** [comment]
HI @jcosham that's awesome! You create the README.Rmd with a skeleton for your content with `use_readme_rmd()`. Then you edit the READM.Rmd file to suit your needs, with code examples, installation instructions, etc., then build the README.md file with `build_readme()`.

**Reply by jcosham** [reply]
Great, thank you Andy! I literally just found and ran this command as you posted - perfect timing to know I'm not going down a dangerous rabbit hole

**Reply by boshek** [reply]
And here is an example of the `Rmd` file @jcosham: https://github.com/boshek/libminer/blob/main/README.Rmd
